app/content-panel/blogs/[id]/page.tsx

```diff
@@ -11,7 +11,7 @@ const BlogDetailsPage = ({ params }: { params: { id: string } }) => {
   const [slug, setSlug] = useState('N/A');
```

Comment:
we are not editing the slug.
slug is the part of a URL that identifies a particular page on a website.
In next.js we use for creating dynamic routes, more information here:https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes

Reply:
Since the slug is used for routing and is not intended for user modification, I should remove the slug field and state entirely from the BlogDetailsCard component,
and should not display the slug in the UI ?

Reply:
@mitkapanarin yes, thats correct.